### PR TITLE
Updated the plugin

### PR DESCRIPTION
**Pull request by Namasteh**
Fixed IMDB
Fixed and improved YouTube
Fixed typo in ReadMe
Added toggle in case there is already a bot with this plugin, now you won't have to unload this plugin or disable it in the bot.rb

### CODE REVIEW

File: lib/cinch/plugins/urlscraper.rb [hunk 1]
```diff
@@ -113,6 +132,29 @@ def listen(m)
...
+        
+        @bot.debug("#{self.class.name} → #{config[:enabled_channels].inspect}");
+        
+      rescue 
+        m.reply Format(:red, "Error: #{$!}")
+      end
     end
   end
 end
+end
```
Comment (mpapis) [inline lib/cinch/plugins/urlscraper.rb]: can you add the new line, I know it's not required, but I prefer to have it

**Comment by mpapis** [comment]
looks great, I will merge as soon as the new line is added

File: lib/cinch/plugins/urlscraper.rb (hunk 1 above)
Comment (Namasteh) [inline lib/cinch/plugins/urlscraper.rb]: I don't understand what you mean by new line, it looks like you're highlighting an end that is already there. I'll be happy to do whatever it is though.

Comment (mpapis) [inline lib/cinch/plugins/urlscraper.rb]: when you go to `Files Changed` tab in the pull request view then you can see a red marker and when you hover over it you get a tool tip `No newline at end of file`.

**Comment by Namasteh** [comment]
Added the newline.

**Comment by Namasteh** [comment]
If/when you update the gem will I be one of the authors?

File: lib/cinch/plugins/urlscraper.rb (hunk 1 above)
Comment (Namasteh) [inline lib/cinch/plugins/urlscraper.rb]: Added the newline

**Comment by mpapis** [comment]
I can add you as an author if you are willing to take care of the plugin, to answer tickets and release new versions

**Comment by Namasteh** [comment]
Oh yeah, I can do that no problem :D

**Comment by mpapis** [comment]
ok you are added now as:
- in gemspec as author in 500bc66
- github collaborator - so you can push code to the repo directly
- gem owner on rubygems - so you can release new gem versions

rules I would like you to follow:
- for easy changes push to `master`
- for bigger changes use branch and open a pull request so we can discuss it
- use semantic versioning
  - for small changes, bugfixes change teeny number
  - for breaking changes bump minor version
  - for big changes / rewrite bump major version
- feel free to add others interested in maintaining the gem

**Comment by mpapis** [comment]
and btw. release time - it's your turn ;) please let me know if you need any help with it, you can find my on irc #rvm or on skype michal_papis, rarely on gtalk on my email

**Comment by Namasteh** [comment]
Okay awesome, sounds great.

**Pull request merged**